applications/ContactStructuralMechanicsApplication/tests/test_ContactStructuralMechanicsApplication.py

```diff
@@ -18,14 +18,18 @@
     missing_application = re.search(r'''.*'KratosMultiphysics\.(.*)'.*''',
                                     '{0}'.format(e)).group(1)
 
+import os
+import sys
+def GetFilePath(fileName):
+    return os.path.join(os.path.dirname(os.path.realpath(__file__)), fileName)
+sys.path.insert(0, GetFilePath('../../../kratos/tests/'))
```

Comment:
inserting things on the path is potentially dangerous, maybe you could use the KratosPaths instead?

Btw does this fix the nightly test-failure?

Reply:
Yep

Reply:
I don't know how to use KratosPaths, I tried yesterday, and this directly worked